src/Azure/Orleans.Clustering.AzureStorage/LegacyAzureMembershipConfigurator.cs

```diff
@@ -16,6 +17,9 @@ public void ConfigureServices(object configuration, IServiceCollection services)
                     options.MaxStorageBusyRetries = reader.GetPropertyValue<int>("MaxStorageBusyRetries");
                     options.ConnectionString = reader.GetPropertyValue<string>("DataConnectionString");
                 });
+
+            services.AddTransient<IConfigurationValidator>(sp => new AzureTableMembershipConfigurationValidator(configuration));
```

Comment:
This and the above IConfigurationValidator change seem orthogonal to this PR - maybe they should be moved into a separate PR

Reply:
Sorry I am new to this.  I thought checking this into my fork would not affect this pull request.  This issue is something else.  When ClusterId is not set you get a Azure Storage Error "Bad Request".  I was had to figure out what was wrong.  This check-in was meant for my branch.

Reply:
No need to apologize :)
You can revert those changes in this branch and they'll disappear from here.

If you'd like assistance, let me know.